diff --git a/main.go b/main.go
index 1e1a835..bb25c53 100644
--- a/main.go
+++ b/main.go
@@ -19,7 +19,7 @@ func main() {
 		Minor: 8,
 		Patch: 3,
 	})
-	nexServer.SetKerberosPassword(os.Getenv("KERBEROS_PASSWORD"))
+	nexServer.SetKerberosPassword(os.Getenv("PN_SPLATOON_KERBEROS_PASSWORD"))
 	nexServer.SetAccessKey("6f599f81")
 
 	nexServer.On("Data", func(packet *nex.PacketV1) {
@@ -35,8 +35,8 @@ func main() {
 
 	secureStationURL := nex.NewStationURL("")
 	secureStationURL.SetScheme("prudps")
-	secureStationURL.SetAddress(os.Getenv("SECURE_SERVER_LOCATION"))
-	secureStationURL.SetPort(os.Getenv("SECURE_SERVER_PORT"))
+	secureStationURL.SetAddress(os.Getenv("PN_SPLATOON_SECURE_SERVER_HOST"))
+	secureStationURL.SetPort(os.Getenv("PN_SPLATOON_SECURE_PORT"))
 	secureStationURL.SetCID("1")
 	secureStationURL.SetPID("2")
 	secureStationURL.SetSID("1")
@@ -48,8 +48,8 @@ func main() {
 	authenticationProtocol.SetPasswordFromPIDFunction(passwordFromPID)
 
 	fmt.Println("==Splatoon - Auth==")
-	fmt.Printf("Secure Server IP: %s\n", os.Getenv("SECURE_SERVER_LOCATION"))
-	fmt.Printf("Secure Server Port: %s\n", os.Getenv("SECURE_SERVER_PORT"))
+	fmt.Printf("Secure Server IP: %s\n", os.Getenv("PN_SPLATOON_SECURE_SERVER_HOST"))
+	fmt.Printf("Secure Server Port: %s\n", os.Getenv("PN_SPLATOON_SECURE_PORT"))
 	fmt.Println("===============")
 
 	nexServer.Listen(":61002")
