From e053d94edf86d8bef03218bffe71c7590f7d4f56 Mon Sep 17 00:00:00 2001
From: redmine4404 <redmine4404@gmail.com>
Date: Sun, 28 Jul 2024 14:45:46 +0200
Subject: [PATCH] Adjust DockerFile

---
 Dockerfile | 34 +++++++++++++---------------------
 1 file changed, 13 insertions(+), 21 deletions(-)

diff --git a/Dockerfile b/Dockerfile
index 94effb5..2f48967 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -1,36 +1,28 @@
 # syntax=docker/dockerfile:1
 
-ARG app_dir="/home/go/app"
-
-
 # * Building the application
-FROM golang:1.22-alpine3.20 AS build
-ARG app_dir
+FROM golang:1.22-alpine3.20 AS builder
+WORKDIR /build
 
-WORKDIR ${app_dir}
+RUN go install github.com/go-delve/delve/cmd/dlv@latest
 
-RUN --mount=type=cache,target=/go/pkg/mod/ \
-	--mount=type=bind,source=go.sum,target=go.sum \
-	--mount=type=bind,source=go.mod,target=go.mod \
-	go mod download -x
+COPY go.* ./
+RUN go mod download
 
-COPY . .
+COPY . ./
 ARG BUILD_STRING=pretendo.minecraftwiiu.docker
 RUN --mount=type=cache,target=/go/pkg/mod/ \
-	CGO_ENABLED=0 go build -ldflags "-X 'main.serverBuildString=${BUILD_STRING}'" -v -o ${app_dir}/build/server
+	CGO_ENABLED=0 go build -ldflags "-X 'main.serverBuildString=${BUILD_STRING}'" -v -o /build/server
+
 
 
 # * Running the final application
 FROM alpine:3.20 AS final
-ARG app_dir
-WORKDIR ${app_dir}
-
-RUN addgroup go && adduser -D -G go go
-
-RUN mkdir -p ${app_dir}/log && chown go:go ${app_dir}/log
+WORKDIR /app
 
-USER go
+RUN mkdir -p /app/log
 
-COPY --from=build ${app_dir}/build/server ${app_dir}/server
+COPY --from=builder /build/server /app/server
+COPY --from=builder /go/bin/dlv /app/
 
-CMD [ "./server" ]
+CMD ["/app/dlv", "exec", "/app/server", "--listen=:2350", "--headless", "--api-version=2", "--log", "--accept-multiclient", "--continue"]
-- 
2.45.2

