diff --git c/.dockerignore i/.dockerignore
new file mode 100644
index 0000000..dd2040f
--- /dev/null
+++ i/.dockerignore
@@ -0,0 +1,3 @@
+.git
+node_modules
+logs
diff --git c/Dockerfile i/Dockerfile
new file mode 100644
index 0000000..233a823
--- /dev/null
+++ i/Dockerfile
@@ -0,0 +1,11 @@
+FROM node:18-alpine
+
+WORKDIR /app
+
+COPY package*.json .
+RUN npm install
+
+COPY . .
+RUN npm run build
+
+CMD ["node", "--inspect=0.0.0.0:9230", "./src/server.js"]
