diff --git i/src/mailer.js w/src/mailer.js
index 283dabc..71d8318 100644
--- i/src/mailer.js
+++ w/src/mailer.js
@@ -1,17 +1,8 @@
 const nodemailer = require('nodemailer');
 const config = require('../config.json');
 
-const transporter = nodemailer.createTransport({
-	host: 'smtp.gmail.com',
-	port: 587,
-	secure: false,
-	auth: {
-		user: config.gmail.user,
-		pass: config.gmail.pass
-	}
-});
-
 async function sendMail(options) {
+	return;
 	options.from = config.gmail.from;
 
 	return await transporter.sendMail(options);
