diff --git a/Dockerfile b/Dockerfile
index 3e08c61..05d9b51 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -9,15 +9,17 @@ ARG app_dir

 WORKDIR ${app_dir}

+RUN go install github.com/go-delve/delve/cmd/dlv@latest
+
 RUN --mount=type=cache,target=/go/pkg/mod/ \
-	--mount=type=bind,source=go.sum,target=go.sum \
-	--mount=type=bind,source=go.mod,target=go.mod \
-	go mod download -x
+    --mount=type=bind,source=go.sum,target=go.sum \
+    --mount=type=bind,source=go.mod,target=go.mod \
+    go mod download -x

 COPY . .
 ARG BUILD_STRING=pretendo.pikmin3.docker
 RUN --mount=type=cache,target=/go/pkg/mod/ \
-	CGO_ENABLED=0 go build -ldflags "-X 'main.serverBuildString=${BUILD_STRING}'" -v -o ${app_dir}/build/server
+    CGO_ENABLED=0 go build -ldflags "-X 'main.serverBuildString=${BUILD_STRING}'" -v -o ${app_dir}/build/server -gcflags "all=-N -l"


 # * Running the final application
@@ -31,6 +33,7 @@ RUN mkdir -p ${app_dir}/log && chown go:go ${app_dir}/log

 USER go

+COPY --from=build /go/bin/dlv ${app_dir}/dlv
 COPY --from=build ${app_dir}/build/server ${app_dir}/server

-CMD [ "./server" ]
+CMD ["./dlv", "exec", "./server", "--listen=:2353", "--headless", "--api-version=2", "--log", "--accept-multiclient", "--continue"]
