diff --git i/src/services/nnid/routes/content.ts w/src/services/nnid/routes/content.ts
index b4d1ac3..9a8b374 100644
--- i/src/services/nnid/routes/content.ts
+++ w/src/services/nnid/routes/content.ts
@@ -39,14 +39,14 @@ router.get('/agreements/:type/:region/:version', (request: express.Request, resp
 						},
 						main_text: {
 							'@index': '1',
-							'#cdata': 'Welcome to Pretendo\'s Christmas public beta! This is supplied with no liability or warranty, and is a stress test of our current services.This test is not expected to last long- term, and the data may be kept for later testing; this data will not be shared outside of Pretendo, and will be deleted at the end of our testing period.'
+							'#cdata': 'Welcome to your selfhosted Pretendo server! Have fun! :)'
 						},
 						sub_title: {
 							'#cdata': 'Privacy Policy'
 						},
 						sub_text: {
 							'@index': '1',
-							'#cdata': 'Welcome to Pretendo\'s Christmas public beta! This is supplied with no liability or warranty, and is a stress test of our current services.This test is not expected to last long- term, and the data may be kept for later testing; this data will not be shared outside of Pretendo, and will be deleted at the end of our testing period.'
+							'#cdata': 'Welcome to your selfhosted Pretendo server! Have fun! :)'
 						},
 					},
 					type: 'NINTENDO-NETWORK-EULA',
@@ -72,14 +72,14 @@ router.get('/agreements/:type/:region/:version', (request: express.Request, resp
 						},
 						main_text: {
 							'@index': '1',
-							'#cdata': 'Welcome to Pretendo\'s Christmas public beta! This is supplied with no liability or warranty, and is a stress test of our current services.This test is not expected to last long- term, and the data may be kept for later testing; this data will not be shared outside of Pretendo, and will be deleted at the end of our testing period.'
+							'#cdata': 'Welcome to your selfhosted Pretendo server! Have fun! :)'
 						},
 						sub_title: {
 							'#cdata': 'Privacy Policy'
 						},
 						sub_text: {
 							'@index': '1',
-							'#cdata': 'Welcome to Pretendo\'s Christmas public beta! This is supplied with no liability or warranty, and is a stress test of our current services.This test is not expected to last long- term, and the data may be kept for later testing; this data will not be shared outside of Pretendo, and will be deleted at the end of our testing period.'
+							'#cdata': 'Welcome to your selfhosted Pretendo server! Have fun! :)'
 						},
 					},
 					type: 'NINTENDO-NETWORK-EULA',
@@ -105,14 +105,14 @@ router.get('/agreements/:type/:region/:version', (request: express.Request, resp
 						},
 						main_text: {
 							'@index': '1',
-							'#cdata': 'Welcome to Pretendo\'s Christmas public beta! This is supplied with no liability or warranty, and is a stress test of our current services.This test is not expected to last long- term, and the data may be kept for later testing; this data will not be shared outside of Pretendo, and will be deleted at the end of our testing period.'
+							'#cdata': 'Welcome to your selfhosted Pretendo server! Have fun! :)'
 						},
 						sub_title: {
 							'#cdata': 'Privacy Policy'
 						},
 						sub_text: {
 							'@index': '1',
-							'#cdata': 'Welcome to Pretendo\'s Christmas public beta! This is supplied with no liability or warranty, and is a stress test of our current services.This test is not expected to last long- term, and the data may be kept for later testing; this data will not be shared outside of Pretendo, and will be deleted at the end of our testing period.'
+							'#cdata': 'Welcome to your selfhosted Pretendo server! Have fun! :)'
 						},
 					},
 					type: 'NINTENDO-NETWORK-EULA',
