diff --git i/src/services/datastore/routes/upload.ts w/src/services/datastore/routes/upload.ts
index 3ceccda..12f853b 100644
--- i/src/services/datastore/routes/upload.ts
+++ w/src/services/datastore/routes/upload.ts
@@ -1,4 +1,3 @@
-import fs from 'node:fs';
 import crypto from 'node:crypto';
 import express from 'express';
 import Dicer from 'dicer';
@@ -6,7 +5,7 @@ import { uploadCDNAsset } from '@/util';
 
 const router: express.Router = express.Router();
 
-const signatureSecret: Buffer = fs.readFileSync(`${__dirname}/../../../../certs/nex/datastore/secret.key`);
+const signatureSecret: Buffer = Buffer.from(process.env.PN_ACT_CONFIG_DATASTORE_SIGNATURE_SECRET || '', 'hex');
 
 function multipartParser(request: express.Request, response: express.Response, next: express.NextFunction): void {
 	const RE_BOUNDARY: RegExp = /^multipart\/.+?(?:; boundary=(?:(?:"(.+)")|(?:([^\s]+))))$/i;
