diff --git i/go.mod w/go.mod
index 5141359..6ed615d 100644
--- i/go.mod
+++ w/go.mod
@@ -2,6 +2,8 @@ module github.com/PretendoNetwork/wiiu-chat-secure
 
 go 1.18
 
+replace github.com/PretendoNetwork/nex-protocols-common-go => ./nex-protocols-common-go
+
 require (
 	github.com/PretendoNetwork/grpc-go v1.0.1
 	github.com/PretendoNetwork/nex-go v1.0.28
diff --git a/Dockerfile b/Dockerfile
index 08da995..f61daae 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -11,6 +11,11 @@ WORKDIR ${app_dir}
 
 RUN go install github.com/go-delve/delve/cmd/dlv@latest
 
+RUN apk add git
+RUN git clone --depth 1 --branch v1.0.22 https://github.com/PretendoNetwork/nex-protocols-common-go
+# Terrible hack to work around the issue where the ticket timestamp is 0
+RUN sed -i 's/ticket\.Timestamp()\.Standard()/time\.Now()\.UTC()/g' nex-protocols-common-go/secure-connection/connect.go
+
 RUN --mount=type=cache,target=/go/pkg/mod/ \
 	--mount=type=bind,source=go.sum,target=go.sum \
 	--mount=type=bind,source=go.mod,target=go.mod \
